src/rust/bitbox02-rust/src/hww/api/show_mnemonic.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate alloc;
+use alloc::vec::Vec;
+
+use super::pb;
+use super::Error;
+
+use pb::response::Response;
+
+use crate::workflow::{mnemonic, status, unlock};
+use bitbox02::keystore;
+
+const NUM_RANDOM_WORDS: u8 = 5;
+
+fn create_random_unique_words(word: &str, length: u8) -> (u8, Vec<&str>) {
+    fn rand16() -> u16 {
```

Comment:
Would it make sense to move this into a util mod?

Reply:
Seems to be too specific to be in general util.